src/components/styled/ResponsiveContainer.jsx

```diff
@@ -0,0 +1,29 @@
+import { styled } from '@mui/material/styles';
+
+const ResponsiveContainer = styled('div')`
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-right: auto;
+  margin-left: auto;
+  box-sizing: border-box;
+  @media (min-width: 600px) {
+    width: 570px;
+  }
+  @media (min-width: 768px) {
+    width: 720px;
+  }
+  @media (min-width: 900px) {
+    width: 870px;
+  }
+  @media (min-width: 1200px) {
+    width: 1170px;
+  }
+  @media (min-width: 1536px) {
+    width: 1400px;
+  }
+  @media (max-width: 599px) {
+    width: 100%;
+  }
```

Comment:
Even though its totally up to you, please consider using common breakpoints. Example:

320px — 480px: Mobile devices
481px — 768px: iPads, Tablets
769px — 1024px: Small screens, laptops
1025px — 1200px: Desktops, large screens
1201px and more —  Extra large screens, TV

or Bootstrap breakpoints:
 xs: 0,
  sm: 576px,
  md: 768px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px

Reply:
all these breakpoints I took from MUI (https://mui.com/material-ui/customization/breakpoints/#main-content), except for 768px, it's needed just for more adaptability

Reply:
Ok, great)